zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -72,30 +97,79 @@
 
   /**
    * note configurations.
-   *
+   * <p/>
```

Comment:
In Zeppelin generally we do not user HTML in JavaDoc, so it's better to be consistent